datacite-doi-commons/src/test/java/no/unit/nva/doi/model/DoiUpdateDtoTest.java

```diff
@@ -0,0 +1,33 @@
+package no.unit.nva.doi.model;
+
+import static no.unit.nva.hamcrest.DoesNotHaveNullOrEmptyFields.doesNotHaveNullOrEmptyFields;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.net.URI;
+import java.time.Instant;
+import nva.commons.utils.JsonUtils;
+import org.junit.jupiter.api.Test;
+
+public class DoiUpdateDtoTest {
+
+    private final ObjectMapper objectMapper = JsonUtils.objectMapper;
+
+    @Test
+    public void canWriteDoiUpdateDtoToJsonAndBack() throws JsonProcessingException {
+        DoiUpdateDto doiUpdateDto = new DoiUpdateDto.Builder()
+            .withDoi(URI.create("http://sample.doi"))
+            .withModifiedDate(Instant.now())
+            .withPublicationId(URI.create("http://sample.publication.id"))
```

Comment:
`sample.publication.id.test`

`.test` is valid domain name to use for testing and wont ever be snapped up and registered by anyone.